index.js

```diff
@@ -70,10 +76,12 @@ module.exports = function(source) {
         styl.define(defineName, value[defineName]);
       }
     } else if (key === 'include') {
-      needsArray(value).forEach(styl.include);
+      needsArray(value).forEach(function(path) {
```

Comment:
Good catch; adding another TODO to set up `eslint`. Right now all the dev tooling is the same as upstream, no linting.

Side note about this particular line, it's fixing a bug in the upstream code. `styl.include` uses `this` (which should be the Stylus renderer instance) but this `forEach` isn't passing `styl` as the context, so it's broken; apparently the `include` option just isn't tested.

We could potentially avoid the `path` naming here entirely by doing `needsArray(value).forEach(styl.include, styl)` instead, although that still relies on knowing that `include` only takes 1 parameter.